python/modules/IcePy/PropertiesAdmin.cpp

```diff
@@ -138,12 +104,13 @@ nativePropertiesAdminRemoveUpdateCB(NativePropertiesAdminObject* self, PyObject*
         return 0;
     }
 
-    for(vector<UpdateCallbackWrapperPtr>::const_iterator p = (*self->callbacks).begin(); p != (*self->callbacks).end();
+    for(vector<pair<PyObject*, function<void()>>>::const_iterator p = (*self->callbacks).begin(); p != (*self->callbacks).end();
```

Comment:
using find_if might be more concise here too

Reply:
Updated. Also added the missing erase.